app/_locales/en/messages.json

```diff
@@ -6404,6 +6404,9 @@
   "transactionFailed": {
     "message": "Transaction Failed"
   },
+  "transactionFailedBannerMessage": {
+    "message": "This transaction would have cost you extra fees, so we stopped it. Your money is still in your wallet."
```

Comment:
Do you think the phrase 'would have cost you extra fees' is clear to users? Even as a developer, I found it a bit ambiguous. Could we consider adding a link to documentation for more context in the future? What do you think, @coreyjanssen?

Reply:
Thanks for sharing your concern Derek, apparently there's a conversation going on with the copy already, you can find in the figma link i shared. It is not obvious actually so I'll share again here 
<img width="804" alt="Screenshot 2024-12-20 at 14 49 52" src="https://github.com/user-attachments/assets/08b8cd10-eea9-42ed-8d9b-ca85bb637310" />